zcash_keys/Cargo.toml

```diff
@@ -67,6 +67,9 @@ byteorder = { workspace = true, optional = true }
 # - Digests
 blake2b_simd = { workspace = true }
 
+# - Error handling
+snafu.workspace = true
```

Comment:
Could `snafu` be a dev dependency instead? If so, I'm amenable to using this instead of `thiserror`, but I would prefer that it not result in an added dependency that could ultimately affect the public API, since `snafu` includes types that should *not* be used in library crates.